src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/lobsTest.java

```diff
@@ -432,7 +431,6 @@ private void testLobsInsertRetrive(String types[], Class lobClass) throws Except
                 while ((read = stream.read(chunk)) > 0)
                     buffer.write(chunk, 0, read);
                 assertEquals(chunk.length, size);
-
```

Comment:
avoid adding new lines if no changes are added (quoted from our onenote discussion 👌 )

Reply:
line was removed because I didn't like the spacing. I think this file is going to have merge conflicts with the Junit PR anyways, will address then.